pkg/scheduler/metrics/metrics.go

```diff
@@ -280,6 +325,7 @@ var (
 		PreemptionVictims,
 		PreemptionAttempts,
 		pendingPods,
+		InFlightEvents,
```

Comment:
It shouldn't be a part of `metricsList`, it should be registered under a feature gate.

Reply:
Removed it from `metricsList`, thanks!